content/post/2017-05-22-ewenme-site.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2017-05-22
+author: Ewen Henderson
+title: Welcome to ewenme (you-n-me)
+tags: 
+ - Hugo
+ - Blogdown
+ - Netlify
+description: An intro to Ewen's new website
+---
+
+## A little bit on me
+
+I'm Ewen, an analyst and data science hopeful with a social sciences background, currently working in UK government. I'm about to join the [Bank of England](http://www.bankofengland.co.uk/Pages/home.aspx), the central bank for the United Kingdom, having been working for [Transport for London](https://tfl.gov.uk/) (government body responsible for the transport system in London) for a while.
+
+## Why start a site?
+
+I've been using R for data science-type workflows pretty intensively for about a year-and-a-half, and I had the realisation a few months back that I felt ready to add something to this conversation. I was also listening a lot to a journalism podcast, [Longform](https://longform.org/podcast), and they made writing sound quite noble/fun. Unfortunately for me, this realisation came before I discovered blogdown, so at first I put together a site using [Github Pages](https://pages.github.com/) and [Jekyll](https://jekyllrb.com/) (boo - the source files will remain over at this [Github repo](https://github.com/ewenme/ewenme.github.io)). 
+
+After coming across Yihui's tour-de-force `Blogdown` package soon after (I think it was in [this rstudio::conf talk](https://www.rstudio.com/resources/videos/advanced-r-markdown-tutorial/)), I knew that jumping across to this set-up sooner rather than later would save me time and make my new blogging life a good deal nicer. So, I wasted no time...
+
+## Going live
+
+My website [ewenme](http://ewenme.rbind.io/) (pronounced you-n-me) is now live, and part of the Rbind family (source files over [here](https://github.com/rbind/ewenme). It's hosted over at [Netlify](https://www.netlify.com/), which (I'm sure everyone has said already) was easy to get going with (and integrate with Git). 
```

Comment:
I don't think `as` is necessary here. "My website is part of the Rbind family" sound correct to me, instead of "My website is as part of the Rbind family".